### PR TITLE
Initial Commit to AMSR

### PR DESCRIPTION
**Pull request by srilathamaddineni**

### CODE REVIEW

File: book/chapters/amsr.ipynb [hunk 1]
```diff
@@ -2,13 +2,918 @@
...
+    "        current_date += delta\n",
+    "\n",
+    "    return links\n",
+    "\n",
+    "if __name__ == \"__main__\":\n",
+    "    start_year = 2019\n",
+    "    end_year = 2022\n",
+    "\n",
+    "    links = generate_links(start_year, end_year)\n",
+    "    save_location = \"C:/Users/srila/gridmet_test_run/amsr\"\n",
```
Comment (ZihengSun) [inline book/chapters/amsr.ipynb]: You shouldn't use your personal directory path in any of the notebooks. Please change it to relative path and make sure it works on all people's laptops.

**Review by ZihengSun — CHANGES_REQUESTED** [review]
Please see my comments.

**Comment by saivivek116** [comment]
- [x] The headings need to be more explanative currently the headings are function names instead you can give meaningful headings

- [ ] As there are several functions in this chapter(divide into sub chapters if needed) its better you try to make it more engaging with headings, explanations and graphs 

- [x] explanation of the code inside function and the purpose of the function can be separated( into different paragraphs) divide the larger function into chunks

- [x] give conclusion of the chapter

- [x] remove unused code
 
- [x] Once the paths are modified and respective files needed to execute the notebook  are uploaded will verify further

**Review by saivivek116 — CHANGES_REQUESTED** [review]
pls check my comments.

**Comment by ZihengSun** [comment]
@saivivek116 please help with this PR by directly contributing to the content.

**Review by ZihengSun — APPROVED** [review]
I am going to merge this PR. @saivivek116 please correct the two left issues in separate PRs.

**Pull request merged**